packages/app-mobile/root.tsx

```diff
@@ -919,7 +930,7 @@ class AppComponent extends React.Component {
 					}}
 				>
 					<StatusBar barStyle={statusBarStyle} />
-					<MenuContext style={{ flex: 1 }}>
+					<MenuProvider style={{ flex: 1 }}>
```

Comment:
I've been getting warnings about `MenuContext` being deprecated. Thank you for changing this!